app/uk/gov/hmrc/tariffclassificationfrontend/controllers/SearchController.scala

```diff
@@ -62,7 +62,8 @@ class SearchController @Inject()(authenticatedAction: AuthenticatedAction,
             attachments: Map[Case, Seq[StoredAttachment]] <- fileStoreService.getAttachments(cases.results)
             results: Paged[SearchResult] = cases.map(c => SearchResult(c, attachments.getOrElse(c, Seq.empty)))
           } yield Results.Ok(html.advanced_search(SearchForm.form.fill(data), Some(results), keywords, selectedTab))
-                            .addingToSession((backToSearchResultsLinkLabel, "search results"), (backToSearchResultsLinkUrl, s"${SearchController.search(selectedTab, None, search, sort, page).url}#advanced_search_keywords"))
+                            .addingToSession((backToSearchResultsLinkLabel, "search results"), (backToSearchResultsLinkUrl,
+                              s"${SearchController.search(selectedTab, None, search, sort, page).url}#advanced_search_keywords"))
```

Comment:
Why is this a String?
Just do `SearchController.search(selectedTab, None, search, sort, page).withFragment("advanced_search_keywords").url`

Reply:
I think we tried that when the tag was added to the search result url but it didn't work for some reason.  Leaving as is for now